File: src/main/java/com/sunset/item/fishes/ItemArowanaFish.java
package com.sunset.item.fishes;

import net.minecraft.world.food.FoodProperties;
import net.minecraft.world.item.Item;

public class ItemArowanaFish extends Item
{
    public ItemArowanaFish() {
        super(GetProperties());
    }

    public static Properties GetProperties() {
        FoodProperties foodProperties = new FoodProperties.Builder()
                .nutrition(3)
                .saturationMod(0.6F)
                .build();
        return new Properties()
                .food(foodProperties)
                .fireResistant();
    }
}

File: src/main/java/club/redux/sunset/lavafishing/item/fishes/ItemArowanaFish.kt
package club.redux.sunset.lavafishing.item.fishes

import net.minecraft.world.food.FoodProperties
import net.minecraft.world.item.Item

class ItemArowanaFish : Item(
    Properties()
        .food(
            FoodProperties.Builder()
                .nutrition(3)
                .saturationMod(0.6f)
                .build()
        )
        .fireResistant()
)